vcs/git/clone_update.go

```diff
@@ -45,7 +45,10 @@ func init() {
 }
 
 func Clone(url, dir string, opt vcs.CloneOpt) (vcs.Repository, error) {
-	clopt := git2go.CloneOptions{Bare: opt.Bare}
+	clopt := git2go.CloneOptions{
+		FetchOptions: &git2go.FetchOptions{},
```

Comment:
Although that wouldn't scale well if more than one `clopt.FetchOptions` field is being optionally set. Perhaps leaving it as is is best.